src/core/truthTable/pla_parser.cpp

```diff
@@ -0,0 +1,91 @@
+#include "core/truthTable/pla_parser.hpp"
+
+namespace syrec {
+
+    [[maybe_unused]] void pla_parser(std::istream& in, syrec::TruthTable& reader) {
+        std::string line;
+        std::regex  whitespace("\\s+");
+
+        while (in.good() && getline(in, line)) {
+            trim(line);
+            line = std::regex_replace(line, whitespace, std::string(" "));
+            if ((line.empty()) || (line.rfind('#', 0) == 0) || (line.rfind(".i", 0) == 0) || (line.rfind(".o", 0) == 0) || (line.rfind(".ilb", 0) == 0) || (line.rfind(".ob", 0) == 0) || (line.rfind(".p", 0) == 0) || (line.rfind(".type ", 0) == 0)) { continue; }
```

Comment:
why does the parser ignore all "frontmatter" declarations? couldn't these contain useful information?

Reply:
To implement our functionality I don't think its required. Later on if we need any additional information, we can use them.